src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/ProjectCapability.cs

```diff
@@ -36,5 +36,6 @@ internal static class ProjectCapability
         public const string ProjectConfigurationsDeclaredDimensions = ProjectCapabilities.ProjectConfigurationsDeclaredDimensions;
         public const string LanguageService = ProjectCapabilities.LanguageService;
         public const string SortByDisplayOrder = ProjectCapabilities.SortByDisplayOrder;
+        public const string DotNet = ".NET";
```

Comment:
Don't you need to remove the one on line 14?

Reply:
yeah that would help.